lib/phoenix_live_view/router.ex

```diff
@@ -45,10 +45,24 @@ defmodule Phoenix.LiveView.Router do
         unquote(path),
         Phoenix.LiveView.Controller,
         Phoenix.Router.scoped_alias(__MODULE__, unquote(live_view)),
-        private: %{phoenix_live_view: unquote(opts)},
+        private: %{
+          phoenix_live_view: unquote(opts),
+          phoenix_live_view_default_layout: Phoenix.LiveView.Router.layout_from_router_module(__MODULE__)
+        },
         as: unquote(opts)[:as] || :live,
         alias: false
       )
     end
   end
+
+  @doc false
+  def layout_from_router_module(module) do
```

Comment:
You should prefix the number with underscores, otherwise it will be imported when the `router` is imported. Usualy we go with `__layout_from_router_module__`.